bot/cogs/off_topic_names.py

```diff
@@ -47,7 +48,6 @@ class OffTopicName(Converter):
             The running bot instance, used for fetching data from the
             website via the bot's `api_client`.
```

Comment:
There doesn't seem to be a consistent way we're listing parameters in docstrings. For example, this is different compared to one of the docstrings in the docs cog. I was under the impression we'd just add a separate paragraph for each parameter akin to Python's own documentation. Such a list seems unnecessary anyway in this case given there's only one argument.